game/src/Scope/QuantumButton.gd

```diff
@@ -0,0 +1,10 @@
+extends Label
+
+var instructions = "This is the quantum button. " \
+	+ "Click here\n to swap between color" \
```

Comment:
er minor thing but it should probably be colour since the name of the game is in British English (the c in defence)

Reply:
tc fixed.